libraries/helpers.rb

```diff
@@ -446,3 +445,6 @@ def apache_mod_php_filename
     end
   end
 end
+
+Chef::Recipe.include Apache2::Cookbook::Helpers
```

Comment:
This is actually the wrong class and I have a cookstyle rule for this, but since you're using the modern .include method it's not triggering. Time to update that. Look at ChefCorrectness/IncorrectLibraryInjection